static/app/icons/iconSentryPride.tsx

```diff
@@ -0,0 +1,15 @@
+import {forwardRef} from 'react';
+
+import {SvgIcon, SVGIconProps} from './svgIcon';
+
+const IconSentryPride = forwardRef<SVGSVGElement, SVGIconProps>((props, ref) => {
+  return (
+    <SvgIcon {...props} ref={ref}>
```

Comment:
We might need to drop or replace the `fill` prop from here if we want a non-gray logo to display.